package.json

```diff
@@ -47,7 +47,7 @@
     "pixi-animate": "^1.3.5",
     "pixi-sound": "^2.1.4",
     "pixi.js": "^4.8.8",
-    "springroll": "^2.0.0",
+    "springroll": "2.1.1",
```

Comment:
I put in an explicit version of SpringRoll since we have a tween issue on the latest version.